alphacsc/tests/test_z_encoder.py

```diff
@@ -3,22 +3,13 @@
 import pytest
 
 from alphacsc._z_encoder import get_z_encoder_for
-from alphacsc.init_dict import init_dictionary
 from alphacsc.loss_and_gradient import compute_objective
 from alphacsc.utils import construct_X_multi
 from alphacsc.utils.compute_constants import compute_ztz, compute_ztX
 
-N_CHANNELS, N_TIMES = 3, 30
-N_TIMES_ATOM, N_ATOMS = 6, 4
-
+from conftest import N_ATOMS, N_TIMES_ATOM
 
-@pytest.fixture
-def D_hat(X, rank1):
-    return init_dictionary(X,
-                           N_ATOMS,
-                           N_TIMES_ATOM,
-                           random_state=0,
-                           rank1=rank1)
+N_CHANNELS, N_TIMES = 3, 30
```

Comment:
Why not import the one from conftest?

Reply:
As they were different, I did not touch the values. I could have asked actually. 
I import from conftest now.